docs/content-services/README.md

```diff
@@ -42,7 +42,7 @@ for more information about installing and using the source code.
 | [Folder create directive](folder-create.directive.md) | Allows folders to be created. | [Source](../../lib/content-services/folder-directive/folder-create.directive.ts) |
 | [Folder edit directive](folder-edit.directive.md) | Allows folders to be edited. | [Source](../../lib/content-services/folder-directive/folder-edit.directive.ts) |
 | [File draggable directive](file-draggable.directive.md) | Provide drag-and-drop features for an element such as a `div`. | [Source](../../lib/content-services/upload/directives/file-draggable.directive.ts) |
-| [Share Directive](share.directive.md) | Proide a pop up that generat a public linlk to share a file | [Source](../../lib/content-services/upload/directives/share.directive.ts) |
+| _Node share directive_ | _Not currently documented_ | [Source](../../lib/content-services/directives/node-share.directive.ts) |
```

Comment:
the link is wrong but we have the documentation https://github.com/Alfresco/alfresco-ng2-components/blob/development/docs/content-services/share.directive.md